Assets/MixedRealityToolkit.Services/InputSystem/FocusProvider.cs

```diff
@@ -772,8 +773,10 @@ private void RaycastGraphics(PointerData pointerData, LayerMask[] prioritizedLay
             }
 
             // Check if we need to overwrite the physics raycast info
-            if ((pointerData.CurrentPointerTarget == null || overridePhysicsRaycast) && raycastResult.isValid &&
-                 raycastResult.module != null && raycastResult.module.eventCamera == UIRaycastCamera)
+            if ((pointerData.CurrentPointerTarget == null || overridePhysicsRaycast)
+                && raycastResult.isValid
```

Comment:
Isn't the usual formatting with the `&&` on the right?

Reply:
Updated